README.md

```diff
@@ -13,4 +13,10 @@ Define and execute a long-term support (LTS) plan for the io.js process.
 
 ## Working Group Members
 
-_To be filled in_
+* Rod Vagg [GitHub/@rvagg](https://github.com/rvagg) / [Twitter/@rvagg](https://twitter.com/rvagg)
+* Ben Noordhuis [GitHub/@bnoordhuis](https://github.com/bnoordhuis) / [Twitter/@bnoordhuis](http://twitter.com/bnoordhuis)
```

Comment:
Can you drop the twitter link?  The account is just a name squatting countermeasure, I don't do (or like) twitter.